stable_baselines/td3/__init__.py

```diff
@@ -0,0 +1,2 @@
+from stable_baselines.td3.td3 import TD3
+from stable_baselines.td3.policies import MlpPolicy, CnnPolicy, LnMlpPolicy, LnCnnPolicy
```

Comment:
I would add 
```python
from stable_baselines.ddpg import noise
```

As such you can use
```python
from stable_baselines.td3.noise import NormalActionNoise
```
for example. It would avoid confusion when importing ddpg when you want to used TD3 I think.

Or even better move the noise classes to somewhere common.

Reply:
>Or even better move the noise classes to somewhere common.

I would agree to move that to common. We need to be sure we won't break thing (so we need to leave the noise file for now in DDPG)

Reply:
> We need to be sure we won't break thing (so we need to leave the noise file for now in DDPG)

You dont need to leave it in DDPG, you can add a reference in the `__init__` of DDPG, so it wont break.

But this can be an other pull request if you prefere